docs/gettingstarted/modfiles.md

```diff
@@ -142,29 +149,34 @@ The `ordering` of two mods may cause a crash due to a cyclic dependency, for exa
 
 ## Mod Entrypoints
 
-Now that the `mods.toml` is filled out, we need to provide an entrypoint for the mod. Entrypoints are essentially the starting point for executing the mod. The entrypoint itself is determined by the language loader used in the `mods.toml`.
+Now that the `neoforge.mods.toml` is filled out, we need to provide an entrypoint for the mod. Entrypoints are essentially the starting point for executing the mod. The entrypoint itself is determined by the language loader used in the `neoforge.mods.toml`.
 
 ### `javafml` and `@Mod`
 
-`javafml` is a language loader provided by NeoForge for the Java programming language. The entrypoint is defined using a public class with the `@Mod` annotation. The value of `@Mod` must contain one of the mod ids specified within the `mods.toml`. From there, all initialization logic (e.g. [registering events][events] or [adding `DeferredRegister`s][registration]) can be specified within the constructor of the class.
+`javafml` is a language loader provided by NeoForge for the Java programming language. The entrypoint is defined using a public class with the `@Mod` annotation. The value of `@Mod` must contain one of the mod ids specified within the `neoforge.mods.toml`. From there, all initialization logic (e.g. [registering events][events] or [adding `DeferredRegister`s][registration]) can be specified within the constructor of the class.
 
 ```java
-@Mod("examplemod") // Must match a mod id in the mods.toml
+@Mod("examplemod") // Must match a mod id in the neoforge.mods.toml
 public class Example {
-  public Example(IEventBus modBus) { // The parameter is the mod-specific event bus, needed e.g. for registration and events
+  // The mod constructor can have the following arguments:
+  // - IEventBus: The mod-specific event bus, needed e.g. for registration and events
+  // - ModContainer/FMLModContainer: The container holding this mod's metadata
+  // - Dist: The physical side this mod is loading on
+  public Example(IEventBus modBus) {
     // Initialize logic here
   }
 }
 ```
 
 :::note
-There must be a 1-to-1 matching of mods in the `mods.toml` file and `@Mod` entrypoints. This means that for every mod defined, there must be exactly one `@Mod` annotation with that mod's id.
+There must be a 1-to-1 matching of mods in the `neoforge.mods.toml` file and `@Mod` entrypoints. This means that for every mod defined, there must be exactly one `@Mod` annotation with that mod's id.
```

Comment:
This statement should be reevaluated, since as of Neo 20.6.40, multiple `@Mod` annotations are possible. See the [relevant Discord announcement](https://discord.com/channels/313125603924639766/1187738270386372740/1236618721100042301).

Reply:
Yeah, I saw that. Need to learn it now too.